modules/ROOT/pages/oas3-support.adoc

```diff
@@ -0,0 +1,72 @@
+= Beta Support for OAS 3.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+Phase 1 of the support for OAS 3.0 is included in Anypoint Exchange, API Community Manager, and API Designer.
+
+Perhaps the most important aspect of this beta support is that it lets you publish OAS 3 specifications to your API community, so that you can share them with partners and customers to help drive their adoption.
+
+If you want to implement or deploy API specifications written in OAS 3, however, during this beta you must first convert them to RAML or OAS 2.
+
+== Beta Features in Anypoint Exchange
+
+* Publish an existing OAS 3 specification.
++
+On the *Publish a new asset* page, select *REST API - OAS* in the *Asset type* field.
+* Share OAS 3 specifications.
+* View documentation that is written into OAS 3 specifications.
+* Publish OAS 3 specifications to Anypoint Community Manager.
+* Share OAS 3 specifications as RAML specifications with Anypoint Studio and API Manager.
++
+.How to convert OAS 3.0 files to RAML for import into Anypoint Studio or API Manager
+image::aex-download.png[The Download option]
++
+. In the top-right corner of Anypoint Exchange, click *Download*.
+. Select *As RAML*.
+. After downloading the file, import it into Anypoint Studio or into API Manager.
++
+[NOTE]
+====
+If you want to share an OAS 3 specification in OAS 2, download your OAS 3 specification as-is, and then convert it to OAS 2 by using a tool such as APIMatic.
+====
+
+== Beta Features in API Community Manager
+
+* Publish OAS 3 specifications to your API community, sharing them with partners and customers to help drive their adoption.
+* View documentation that is written into OAS 3 specifications.
+
+== Beta Features in API Designer
+
+* Create API specifications in OAS 3.
++
+When creating a new API-specification project, select *OAS 3.0 (Beta)* in the *Language* field.
+* Import OAS 3 specifications from Anypoint Exchange or from your file system.
++
+.How to import files into an API-specification project in API Designer
+image::apid-import.png[Options for importing]
++
+. Click the gear icon in the top-right corner of the text editor.
+. Select either of these options:
+** Select *Import from Exchange* to see a list of the API specifications that are available from the business organization that your user ID belongs to in Anypoint Platform and a list of the API specifications that are published by MuleSoft.
+** Select *Import* to import an API specification from your local filesystem.
+* Develop and edit OAS 3 specfications.
+* View documentation that is written into OAS 3 specifications.
+* Simulate calls to methods that are in OAS 3 specifications.
+* Publish OAS 3 specifications to Anypoint Exchange.
+* Share OAS 3 specifications as RAML specifications with Anypoint Studio and API Manager.
++
+.How to convert OAS 3.0 files to RAML for import into Anypoint Studio or API Manager
```

Comment:
Let me know your opinion on moving all the conversions under a heading, making it easy for users to understand the options for E2E flows. Something like
== Conversions:
Users can convert between OAS3 specs and RAML 
Using the "duplicate as" option in the API Designer and set the converted spec as the root file before you publish. You can import OAS3 spec into Exchange and use the "download as" feature to convert the spec to RAML and upload it to the Exchange.
Note: The conversions between the languages are the best effort, as there are language-specific features that are not directly supported in other languages and vice-versa. For example, RAML concepts like Resourcetypes, traits, and overlays are not available in OAS3.

Reply:
@harshakirank , can't users convert between OAS3 and OAS2?

Reply:
@harshakirank , I don't think this section makes it easier for users to understand E2E flows because the context in which the conversion takes place is removed. Also, it seems as though you are missing the scenario of downloading as OAS3 from APID for importing specs into Studio or API Manager.